compute/gpu-instances.rst

```diff
@@ -107,7 +108,7 @@ Upgrading the driver
 --------------------
 From time to time, nVidia will release (and Binero will provide) and upgraded version of the GPU driver. This is in order to correct potential bugs and keep the software secure. When this happens, Binero strongly recommends (and in some cases, it will be required to maintain a working system) that the driver be updated on the instances running it. Please follow below instructions to upgrade the driver.
 
-The current version of the driver that we support is: Windows: **537.70** Linux **535.129.03**.
+The latest version of the driver that we support is ** Linux: 535.161.05 Windows: 538.33.**
```

Comment:
says 535.161.05 but the download says 535.161.07

Reply:
Thanks, 

Have fixed and pushed new correct version.